tests/common.py

```diff
@@ -534,7 +534,17 @@ def mock_registry(
     hass: HomeAssistant,
     mock_entries: dict[str, entity_registry.RegistryEntry] | None = None,
 ) -> entity_registry.EntityRegistry:
-    """Mock the Entity Registry."""
+    """Mock the Entity Registry.
```

Comment:
Is it useful to rename the function, and add a compatibility alias?
```python
def mock_entity_registry(...):
    ...
# deprecated - kept for compatibility
mock_registry = mock_entity_registry
```

Reply:
I suggest just renaming that in a separate PR in all test cases, as in, we don't have to keep backward compatibility here. (Besides, most test cases are using it, while they they shouldn't :D)